database-cleanup/database_cleanup/app.py

```diff
@@ -10,6 +10,9 @@
 import os
 import time
 import uuid
+import sentry_sdk
+import pkg_resources
+my_version = pkg_resources.get_distribution('forecast_inference').version
```

Comment:
doesnt this one need to be database-cleanup

Reply:
did you change the wrong one? 

Note your imports all need to be at the top of the file, and then this line needs to be further down. 
You can use tools like `isort` to do this automatically